UPLOADING.md

```diff
@@ -65,10 +65,31 @@ At this point, there should be **no further changes in your code**.
 
 Run the following command **in a folder that does not include your data-loading script**:
 
+Test both the original dataset schema/config and the bigbio schema/config. 
+
+```python
+from datasets import load_dataset
+
+dataset_orig = load_dataset("bigscience-biomedical/<your_dataset_name>", name="source", use_auth_token=True)
+dataset_bigbio = load_dataset("bigscience-biomedical/<your_dataset_name>", name="bigbio", use_auth_token=True)
+```
+
+For datasets that are licensed such that they must already exist on the users local machine, 
+the test commands might look something like this instead,
```

Comment:
@galtay Can you provide an example of the case where "datasets that are licensed such that they must already exist on the users local machine"

Reply:
here is an example of a dataset that cannot be publicly downloaded but we have a dataset loader script that the user can use and point to their local copy of data. https://github.com/bigscience-workshop/biomedical/blob/master/examples/n2c2_2011_coref.py ... In the README FAQs this is is covered under the 
```
What if my dataset does not have a public license?

We understand that some biomedical datasets require external licensing. To respect the agreement of the license, we recommend implementing a dataloader script that works if the user has a locally downloaded file
```
eventually we should make sure that this part of the FAQs points to an up to date, but also simple example and template and make clear how the use of `data_dir` and `data_files` keyword arguments are supported in the `datasets` package. 
See the `Note` in the `datasets` docs for a bit more background https://huggingface.co/docs/datasets/add_dataset.html#downloading-data-files-and-organizing-splits